python/cuml/solvers/qn.pyx

```diff
@@ -541,7 +531,9 @@ class QN(Base):
             del state['handle']
         if 'coef_' in state:
             if state['coef_'] is not None:
-                state['coef_'] = cudf.Series(state['coef_'].ravel())
+                state['coef_'] = cudf.Series(
```

Comment:
this conversions in getstate and setstate are no longer needed, could you remove them? CumlArray (and the handle for that matter) are now pickleable.

Reply:
You're right! I removed them.